### ISSUE TITLE
BUG : remontée des zones REg dans la recherche

### ISSUE THREAD

**Issue by AdelineCelier** [MTES-MCT/monitorfish#56]
Il semble qu'il y ait des couches qui ne remontent pas, ou alors les couches ne correspondent pas à la recherche.
Exemple ici avec MEMN et drague : je ne sais pas quelles sont ces couches qui remontent (couches CSJ ? mais sachant que dans QGIS elles s'affichent juste en "pêche ouverte/pêche fermée". 

![{645078EB-46B1-4360-BAA6-06A7978567ED} png](https://user-images.githubusercontent.com/71878648/103668320-460ea700-4f77-11eb-93c4-e12d32bdeca2.jpg)

**Comment by louptheron** [comment]
Sur la zone MEMN il y a bien de la drague : 
![Screenshot from 2021-01-06 10-47-27](https://user-images.githubusercontent.com/14853556/103754399-97b54100-500c-11eb-840e-1440b1b3b9af.png)

**Comment by AdelineCelier** [comment]
Yes mais il manque des zones ! comme "Moule baie de seine", ou "Bivalves ouest cotentin", ou Crevettes blanches Manche est"

**Comment by louptheron** [comment]
Done, wdyt ?
